lib/spack/spack/build_systems/cmake.py

```diff
@@ -32,6 +32,50 @@
 from spack.package import PackageBase, run_after
 
 
+def processCMakeCacheFile(cache_file):
```

Comment:
Yeah that works.  Can we use `the_same_naming_convention_other_spack_functions_use` instead of `yourFavoriteOne`?  Also I would call this `read_cmake_cache_file`.  Does it use regexes now?